packages/hardhat-core/src/internal/hardhat-network/provider/node-types.ts

```diff
@@ -44,7 +44,7 @@ export interface GenesisAccount {
 }
 
 export interface CallParams {
-  to: Buffer;
+  to?: Buffer;
```

Comment:
This was incorrect. We were using an empty Buffer to signal deployments, and that gave problems with v5.